commands

```diff
@@ -50,6 +50,12 @@ case "$1" in
     dokku_log_verbose_quiet "Waiting for container to be ready"
     docker run --rm --link "$SERVICE_NAME:$PLUGIN_COMMAND_PREFIX" dokkupaas/wait > /dev/null
 
+    if [[ "$3" == "--oldSchema" ]]; then
```

Comment:
My merge-fu is not strong enough... ;)

Helpful stackoverflow post: https://stackoverflow.com/questions/255898/how-to-iterate-over-arguments-in-bash-script